spec/feature/homepage_spec.rb

```diff
@@ -0,0 +1,26 @@
+RSpec.describe "Homepage", type: :feature do
+  before { stub_content_store_has_item("/", schema: "special_route") }
```

Comment:
We'll be adding a schema example for the homepage content item soon so can update this test then.

Reply:
Cool! :)